### ISSUE TITLE
segmentation fault on close client

### ISSUE THREAD

**Issue by theltybc** [grpc/grpc-node#490]
### Problem description
segmentation fault, if close client before finely client.waitForReady.

### Reproduction steps
1. Create and run client, server(or only client).
2. Call client.waitForReady
3. Wait(1ms-10ms)
4. Call client.close
5. Catch segfault

Repository with example: https://github.com/theltybc/example_for_issue_grpc.git

### Environment
 - OS name, version and architecture: Ubuntu mate 16.04.4 LTS (Xenial Xerus) amd64
 - Node version - 9.2.0
 - Node installation method - from site
 - Package version [1.13.0, 1.14.0]

### Additional context
waitForReady call methods of client.$channel. Any method of channel cause an segfault

**Comment by nicolasnoble** [comment]
Crash confirmed. Stacktrace:

```
#0  0x00007ffff48591a0 in grpc_channel_stack_last_element (channel_stack=0x60) at ../deps/grpc/src/core/lib/channel/channel_stack.cc:84
#1  0x00007ffff4838056 in grpc_channel_check_connectivity_state (channel=0x0, try_to_connect=1) at ../deps/grpc/src/core/ext/filters/client_channel/channel_connectivity.cc:37
#2  0x00007ffff47fa345 in grpc::node::Channel::GetConnectivityState (info=...) at ../ext/channel.cc:285
```

Looks like there's a bunch of functions in `ext/channel.cc` that can call a method on a channel without verifying if it's `nullptr` first.

**Comment by murgatroid99** [comment]
I have published the linked fix in version 1.14.1.
